test/qtiCommonRenderer/interactions/textEntry/test.js

```diff
@@ -141,10 +137,6 @@ define([
                     'the container contains a text entry interaction .qti-textEntryInteraction'
                 );
 
-                $input.val('');
-                $input.focus();
-                assert.equal(getTooltipContent($input), __('This is not a valid answer'));
```

Comment:
don't remove this, just change assert to check theres is no tooltip
`assert.ok(!getTooltip($input), 'the error tooltip is hidden after a correct response');`

Reply:
getTooltip always return instance (always true), that don't works. I'm trying different combinations like evaluate against `undefined` or `getTooltip($input).not(':visible')` but still issues